### PR TITLE
[sc-19608] Multistep_GA_docs

### PR DESCRIPTION
**Pull request by drakirnosslin**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

### CODE REVIEW

File: site/content/docs/_index.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ monitoring in minutes.
 {{< doc-card
 	  class="two-column-card"
 	  headerTag="h3"
-	  title="Create a multistep API check"
+	  title="Create a multistep check"
```
Comment (maxigimenez) [inline site/content/docs/_index.md:68]:
```suggestion
	  title="Create a Multistep check"
```

File: site/content/docs/monitoring/check-pricing.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ When a check is [retried](/docs/alerting-and-retries/retries), this counts as a
 [Multistep check](/docs/multistep-checks) pricing is slightly different from browser and API checks. A multistep check is billed based on the number of requests done per check run. Each request in a multistep check run is billed as a single regular API check run, as they are performing the same basic operation. 
 
 {{< info >}}
-**Example:** Let’s say you have 4 API checks, where each check doing one of the `GET`, `POST`, `PUT` and `DELETE` operations towards the same endpoint. If you replace these 4 checks with a single multistep API check that runs 4 requests towards the same endpoint, checking each method, and the check run frequency is the same as before, your cost stays the same.
+**Example:** Let’s say you have 4 API checks, where each check doing one of the `GET`, `POST`, `PUT` and `DELETE` operations towards the same endpoint. If you replace these 4 checks with a single multistep check that runs 4 requests towards the same endpoint, checking each method, and the check run frequency is the same as before, your cost stays the same.
```
Comment (maxigimenez) [inline site/content/docs/monitoring/check-pricing.md:61]:
```suggestion
**Example:** Let’s say you have 4 API checks, where each check doing one of the `GET`, `POST`, `PUT` and `DELETE` operations towards the same endpoint. If you replace these 4 checks with a single Multistep check that runs 4 requests towards the same endpoint, checking each method, and the check run frequency is the same as before, your cost stays the same.
```

File: site/content/docs/monitoring/check-results.md [hunk 3]
```diff
@@ -20,27 +20,21 @@ The summary at the top of the page allows for filtering based on the page's data
...
 <img class="screenshot-partial" alt="check results overview summary" src="/docs/images/monitoring/check-overview-summary.png"/>
 
-Based on the user's selection, the metrics in the summary will also be updated to show the most important numbers at a glance. Single tabs in the summary can be clicked to skip to the related section of the page showing a more detailed breakdown of the relevant information.
+Based on the user's selection, the metrics in the summary will also be updated to show the most important numbers at a glance.
 
-### Time ribbon
+### Monitoring results chart
 
-Check run results can be accessed by selecting them on the time ribbon or by hovering dots and clicking View Details. View Details will open the selected check run in a new browser tab; a simple click on a time ribbon dot will show a preview of the selected check run below it.
+The monitoring results chart shows a summary of the run results in the selected time period where each bar representing a part of that time period. You can change the time period in the summary section.
```
Comment (maxigimenez) [inline site/content/docs/monitoring/check-results.md:27]:
```suggestion
The monitoring results chart shows a summary of the run results in the selected time period where each bar represents a part of that time period. You can change the time period in the summary section. 
```

File: site/content/docs/monitoring/check-results.md [hunk 4]
```diff
@@ -125,6 +122,21 @@ These include:
...
 
+## Multistep check results
+
+Multistep check results are navigated using the tree on the left side of the screen. If you are running checks in parallel, first select the location you are interested in.
+
+In the result tree the top node shows the check run log and the check run configuration. Both can be collapsed.
+
+Any errors encountered can be viewed in the 'Errors' node. 
+
+Each Playwright request done is shown as a separate node under the test step in which is was performed. Selecting a request node opens the request details. Here you can view the request and response body, headers and any request parameters. A breakdown of the request timings is also available.
```
Comment (maxigimenez) [inline site/content/docs/monitoring/check-results.md:133]:
```suggestion
Each Playwright request done is shown as a separate node under the test step in which it was performed. Selecting a request node opens the request details. Here you can view the request and response body, headers and any request parameters. A breakdown of the request timings is also available.
```

Comment (maxigimenez) [reply site/content/docs/monitoring/check-results.md:133]: Also, what about including a small phrase about axios & https requests?

File: site/content/docs/multistep-checks/degraded-state.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+title: Degraded state and soft assertions
+weight: 19
+menu:
+  resources:
+    parent: "Multistep checks"
+    identifier: "multistep-degraded-state-soft-asserts"
+cli: true
+---
+
+If you want to monitor your API for non-critical errors or performance degradations you can use the degraded check state. This allows you to signal that parts of a multistep check performed slower than expected, or that it triggered assertions that are of lower criticality.
```
Comment (maxigimenez) [inline site/content/docs/multistep-checks/degraded-state.md:11]:
```suggestion
If you want to monitor your API for non-critical errors or performance degradations you can use the degraded check state. This allows you to signal that parts of a Multistep check performed slower than expected, or that it triggered assertions that are of lower criticality. 
```

**Review by maxigimenez — CHANGES_REQUESTED** [review]

File: site/content/docs/_index.md (hunk 1 above)
Comment (drakirnosslin) [reply site/content/docs/_index.md:68]: Scrap that, I talked to Sara & she agreed that we should use upper case. I'll change it for the other checks as well on this page.

File: site/content/docs/multistep-checks/degraded-state.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
+---
+title: Degraded state and soft assertions
```
Comment (maxigimenez) [inline site/content/docs/multistep-checks/degraded-state.md:2]: `and soft assertions` sounds like documentation about it, instead is more about the use of it for degraded states, wdyt?

```suggestion
title: Degraded state with soft assertions
```

File: site/content/docs/multistep-checks/degraded-state.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+    if (foo.length > 100) {
+      markCheckAsDegraded('Foo is too long.')
+    }
+```
+
+**Arguments**
+- `reason` String *(optional)*. Logged when the method is called. Used to identify which method caused the degradation.
+
+### getAPIResponseTime
+Gets the reqeust response time.
```
Comment (maxigimenez) [inline site/content/docs/multistep-checks/degraded-state.md:45]:
```suggestion
Gets the request response time.
```

**Review by maxigimenez — CHANGES_REQUESTED** [review]

File: site/content/docs/cli/constructs-reference.md [hunk 8]
```diff
@@ -277,10 +277,10 @@ raw JavaScript / TypeScript as a string.
 
 ## `MultiStepCheck`
 
-Similar to Browser Checks, Multistep API checks uses [`@playwright/test`](https://playwright.dev/) to define the script which the check runs, but Multistep checks always need to be defined in a construct before assigning a `spec.js|ts` file.
+Similar to Browser Checks, Multistep checks uses [`@playwright/test`](https://playwright.dev/) to define the script which the check runs, but Multistep checks always need to be defined in a construct before assigning a `spec.js|ts` file.
```
Comment (miliberlin) [inline site/content/docs/cli/constructs-reference.md:280]:
```suggestion
Similar to Browser checks, Multistep checks uses [`@playwright/test`](https://playwright.dev/) to define the script which the check runs, but Multistep checks always need to be defined in a construct before assigning a `spec.js|ts` file.
```

**Comment by maxigimenez** [comment]
@drakirnosslin the check overview image is outdated

File: site/static/docs/images/monitoring/check-overview.png
Comment (miliberlin) [inline site/static/docs/images/monitoring/check-overview.png:1]: @drakirnosslin can you update chrome and take this screenshot again? 😅

Comment (drakirnosslin) [reply site/static/docs/images/monitoring/check-overview.png:1]: Done!

**Review by maxigimenez — APPROVED** [review]

**Pull request merged**